tests/py/test_email.py

```diff
@@ -46,6 +48,11 @@ def test_participant_can_add_email(self):
         response = self.hit_email_spt('add-email', 'alice@example.com')
         assert response.text == '{}'
 
+    def test_participant_can_add_email_with_special_chars(self):
+        punycode_email = 'alice@' + u'accentué.com'.encode('idna').decode()
+        self.hit_email_spt('add-email', u'alice@accentué.com')
+        assert self.alice.get_any_email() == punycode_email
```

Comment:
Can we see special chars in the username part as well? What happens if we pass in `alïçê@accentué.com`?

Reply:
It should normally throw an error, as this adress doesn't respect the regular expression we are using for validation. If Mangopay doesn't fully support accented emails, I think there is no way to do it on our side.

Reply:
@Bat41 Can we encode that knowledge in a test? :)